utils/xmtpRN/sync.ts

```diff
@@ -128,8 +128,8 @@ const syncClientConversationList = async (account: string) => {
     });
     // Prefetch the conversation list so when we land on the conversation list
     // we have it ready, this will include syncing all groups
-    const conversationList = await fetchConversationListQuery(account);
-    subscribeToNotifications({ conversations: conversationList, account });
+    setupAccountTopicSubscription(account);
+    await prefetchConversationListQuery(account);
```

Comment:
Super good for now, just wonder if later we'll centralize this else where. I feel having this in `xmtpRN/sync.ts` isn't good? 🤔 

Maybe once we refactor the whole account store setup

Reply:
Yeah I definitely this this will be removed, but for now want it